src/compiler/sys.ts

```diff
@@ -556,6 +559,14 @@ namespace ts {
                     return process.cwd();
                 },
                 readDirectory,
+                getModifiedTime(path) {
+                    return _fs.existsSync(path) && _fs.statSync(path).mtime;

```

Comment:
I considered this, but I was thinking that if for some reason `statSync` returned `undefined`, we'd want to know about it. Thinking about it now, I think it might be better to return `undefined` unless someone else has other thoughts.